app/src/main/java/com/jerboa/ui/components/common/Route.kt

```diff
@@ -62,7 +62,7 @@ object Route {
 
         companion object {
             const val ID = "id"
-            val ID_TYPE = NavType.IntType
+            val ID_TYPE = NavType.LongType
```

Comment:
Dang compiler not finding this one

![](https://c.tenor.com/rwmBSrpTcJUAAAAC/tenor.gif)

Reply:
Yeah it sucks we cant do this type safe

Reply:
Oh well. I'll get another release out tmrw with these two fixes.